draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -265,7 +259,6 @@ the implementation first found in BIND 9.11.
 ## Member Zone Properties {#properties}
 
 Each member zone MAY have one or more additional properties, described in this chapter.
```

Comment:
```suggestion
Each member zone MAY have one or more additional properties, described in this chapter.
All member properties described in this document are optional and not mandatory to implement.
```
I understand this follows from the word "MAY", but I feel a bit extra clarification is in place here.